### DISCUSSION TITLE
Is there support for devcontainers?

### DISCUSSION THREAD

**Discussion by lattice0** [coder/code-server#6673] [Q&A]
I'd like to use this for running devcontainers

**Comment by code-asher** [comment]
No, the containers plugin is closed-source and only works with desktop VS Code unfortunately.

**Reply by bilogic** [reply]
what about this? https://open-vsx.org/extension/DDorch/codium-devcontainer

**Reply by code-asher** [reply]
Ooo that looks promising.  It might still not work with code-server because code-server itself is using a remote protocol and I am not sure you can chain remotes, but worth a shot.
